cluster.go

```diff
@@ -1195,16 +1195,9 @@ func (c *ClusterClient) defaultProcessPipeline(cmds []Cmder) error {
 }
 
 func (c *ClusterClient) mapCmdsByNode(cmds []Cmder) (map[*clusterNode][]Cmder, error) {
-	state, err := c.state.Get()
-	if err != nil {
-		setCmdsErr(cmds, err)
-		return nil, err
-	}
```

Comment:
This block is here so all cmds are mapped using single cluster state. Otherwise cluster can be re-configured during mapping and commands can be mapped inconsistently.